make_site.go

```diff
@@ -38,22 +40,28 @@ func main() {
 		panic(err)
 	}
 
-	objs := []Object{}
-	query.Find("body #content ul ul").First().Each(func(_ int, s *goquery.Selection) {
-
+	objs := make(map[string]*Object)
+	query.Find("body #contents").NextFiltered("ul").Find("ul").Each(func(_ int, s *goquery.Selection) {
```

Comment:
Instead of taking the first `ul ul` combination on the page, the adapted query looks for the `Contents` headline and takes all lists in the next `ul` (which is the whole contents list, including "Resources").